lib/Filters.js

```diff
@@ -7,15 +7,17 @@ class Filters extends Node {
   }
 
   toStringStart () {
-    return (this.children.length !== 0) && 'FILTER ('
+    if (this.children.length !== 0) return 'FILTER ('
+    return ''
   }
 
   toStringChildren () {
     return this.children.map(child => space(2) + child.toString()).join(' &&\n')
   }
 
   toStringEnd () {
-    return (this.children.length !== 0) && ')'
+    if (this.children.length !== 0) return ')'
```

Comment:
Can you change it like this?

```javascript
    if (this.children.length !== 0) {
      return ')'
    }
```

That would make it more obvious that this line may exit the function.

Reply:
I can change this but we wouldn't have to discuss individual linting details if we had a proper linting config. IMO we should either use standard or use better, stricter rules, instead of using standard + unwritten rules.